Jenkinsfile

```diff
@@ -58,6 +58,9 @@ stage('prep') {
         }
       }
     }
+    lines.each {
+      stash name: it, includes: "pct.sh,excludes.txt,target/pct.jar,target/megawar-$it.war"
```

Comment:
99% of the stash is the megawar (now that PCT has been slimmed down considerably), so it did not seem worth the bother to separate the generic from line-specific contents. Anyway on S3 there is some fixed overhead per blob operation, and each Pipeline step adds a bit of overhead as well, so minimizing the count is probably the better choice. Also this is more legible I think.

Reply:
make sense 👍